labelbox/schema/asset_attachment.py

```diff
@@ -65,3 +66,29 @@ def delete(self) -> None:
                     id}
             }"""
         self.client.execute(query_str, {"attachment_id": self.uid})
+
+    def update(self,
+               name: Optional[str] = None,
+               type: Optional[str] = None,
+               value: Optional[str] = None):
+        """Updates an attachment on the data row."""
```

Comment:
I belive this goes into our online documentation on [readthedoc](https://readthedocs.org/projects/labelbox-python/) so we should not refer here to any internal gql stuff, just what the user should see

BTW we use sphinx to generate documentation and here is the markup https://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html there is also a structured way to call out args etc some people have been doing

This is optional, we do not enforce this at the moment